src/calendar/date/CalendarUtils.ts

```diff
@@ -208,13 +208,16 @@ export function calculateAlarmTime(date: Date, interval: AlarmInterval, ianaTime
 		.toJSDate()
 }
 
+/** takes a date which encodes the day in UTC and produces a date that encodes the same date but in local time zone. All times must be 0. */
```

Comment:
should we have assertion that the times actually are zero?

Reply:
maybe